sgt-back-end/server.js

```diff
@@ -0,0 +1,146 @@
+import express from './node_modules/express/index.js';
+import { writeFile } from 'node:fs/promises';
+import { readFile } from 'node:fs/promises';
+
+import pg from 'pg';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+
+async function newJsonData() {
+  return JSON.parse(await readFile("./data.json"));
+}
+
+async function creation(data) {
+  return await writeFile("./data.json", JSON.stringify(data, null, 2), 'utf8');
+}
```

Comment:
Oh, it looks like these are leftover from the prior exercise. Delete them since they aren't used.